src/components/Dropdown/Items/MultiSelectItem/MultiSelectItem.tsx

```diff
@@ -3,57 +3,90 @@ import {
     DropdownMenuCheckboxItem,
     DropdownMenuCheckboxItemProps,
 } from '@radix-ui/react-dropdown-menu';
+import { PrioritySelector, PrioritySelectorProps } from '../../../PrioritySelector';
 import { bem } from '../../../../utils';
 import { VisualCheckbox } from '../../../Checkbox';
 import { Text } from '../../../Text';
 import styles from './MultiSelectItem.scss';
 
-export interface Props extends DropdownMenuCheckboxItemProps {
+export interface Props<PriorityItemValue> extends DropdownMenuCheckboxItemProps {
     /** Id for the checkbox */
     id?: string;
     /** a variant to determine the look and feel of component */
     variant?: 'option' | 'select-all' | 'group-title';
     /** to add a priority badge before the label */
     hasPriority?: boolean;
```

Comment:
I wonder if we need this prop. I think it was a placholder only. Maybe we can just assume that if `priority` is defined, then it should be rendered. What do you think?

Reply:
I agree. I removed this at some point :')